### PR TITLE
ZEPPELIN-509 Introduce a new REST API: get notebook information

### PR DESCRIPTION
**Pull request by HeartSaVioR**
### What is this PR for?

To add a new feature - "get notebook information" via REST API.

### What type of PR is it?

Feature

### Is there a relevant Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-509

### How should this be tested?

Just request ```http://<zeppelin host>:<zeppelin port>/api/notebook/<notebookId>```.
If it exists, response would be http status 200 with json body including paragraphs information.
If it doesn't exist, response would be http status 404.

### Questions:
* Does the licenses files need update? (No)
* Is there breaking changes for older versions? (No, it's a new feature.)
* Does this needs documentation? (Yes, I've addressed it.)

### CODE REVIEW

File: docs/rest-api/rest-notebook.md [hunk 1]
```diff
@@ -106,6 +106,288 @@ limitations under the License.
...
+      <td> 500 </td>
+    </tr>
+    <tr>
+      <td> sample JSON response </td>
+      <td><pre>
+{
+  "status": "OK",
+  "message": "",
+  "body": {
+    "paragraphs": [
```
Comment (HeartSaVioR) [inline docs/rest-api/rest-notebook.md]: Please let me know if content of "paragraphs" is too long to display.

**Comment by HeartSaVioR** [comment]
Ready to review.

File: docs/rest-api/rest-notebook.md (hunk 1 above)
Comment (Leemoonsoo) [inline docs/rest-api/rest-notebook.md]: Right, it looks bit long.

Comment (HeartSaVioR) [inline docs/rest-api/rest-notebook.md]: I just keep information of the first paragraph and remove others. Please let me know if it is still too long.

**Comment by Leemoonsoo** [comment]
CI failing is irrelevant to this PR.
LGTM. 
Merge if there're no more discussions.

**Pull request closed without merging**